docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -369,14 +369,35 @@ spring:
   cloud:
     gateway:
       routes:
-      - id: add_request_header_route
+      - id: add_response_header_route
         uri: http://example.org
         filters:
         - AddResponseHeader=X-Response-Foo, Bar
 ----
 
 This will add `X-Response-Foo:Bar` header to the downstream response's headers for all matching requests.
 
+=== DedupeResponseHeader GatewayFilter Factory
+The DedupeResponseHeader GatewayFilter Factory takes a name parameter and an optional strategy parameter.
+
+.application.yml
+[source,yaml]
+----
+spring:
+  cloud:
+    gateway:
+      routes:
+      - id: dedupe_response_header_route
+        uri: http://example.org
+        filters:
+        - DedupeResponseHeader=Access-Control-Allow-Credentials
```

Comment:
Could we make the filter take in a list of headers rather than specifying the filter for each header?